tests/helpers/perf_run_helper.rb

```diff
@@ -564,9 +568,9 @@ def check_scale_results(scenario_hash)
     # add this row to the csv
     update_scale_results_csv(scale_results_parent_dir, results)
 
-    # this needs to be last
-    if num_ko != 0
-      puts "ERROR - KO encountered in scenario: #{scenario}"
+    # allow no more than 10 KOs per iteration; this needs to be last
+    if num_ko > 10
```

Comment:
Please use a constant for this value to prevent the proliferation of magic numbers.
Suggestion
```
MAX_ALLOWED_KO = 10
if num_ko > MAX_ALLOWED_KO
  puts "ERROR - more than #{MAX_ALLOWED_KO} KOs encountered in scenario: #{scenario}"
```

Reply:
Great suggestion! Updated...